openedx/features/edly/utils.py

```diff
@@ -25,3 +30,22 @@ def decode_edly_user_info_cookie(encoded_cookie_data):
         dict
     """
     return jwt.decode(encoded_cookie_data, settings.EDLY_COOKIE_SECRET_KEY, algorithms=[settings.EDLY_JWT_ALGORITHM])
+
+
+def get_enabled_organization(request):
+    """
+    Helper method to get linked organizations for request site.
+
+    Returns:
+        list: List of linked organizations for request site
+    """
+
+    if not waffle.switch_is_active(settings.ENABLE_EDLY_ORGANIZATIONS_SWITCH):
+        organizations = get_organizations()
```

Comment:
import logging and set logger at the start of the file:

`LOGGER = logging.getLogger(__name__)`

```python
if not waffle.switch_is_active(settings.ENABLE_EDLY_ORGANIZATIONS_SWITCH):
    return get_organizations()

try:
    studio_site_edx_organization = request.site.studio_site.edx_organization
except EdlySubOrganization.DoesNotExist:
    LOGGER.exception('No EdlySubOrganization found for site %s', request.site)
    return []

return [studio_site_edx_organization]
```

Also we will need to add test for exception log message if there is no related `EdlySubOrganization` model for site.

Reply:
not implemented